docs/data-sources/instance_servers.md

```diff
@@ -44,8 +44,12 @@ In addition to all above arguments, the following attributes are exported:
         ~> **Important:** Instance servers' IDs are [zoned](../guides/regions_and_zones.md#resource-ids), which means they are of the form `{zone}/{id}`, e.g. `fr-par-1/11111111-1111-1111-1111-111111111111`
 
     - `tags` - The tags associated with the server.
-    - `public_ip` - The public IPv4 address of the server.
+    - `public_ip` - The public IP address of the server.
     - `private_ip` - The Scaleway internal IP address of the server.
+    - `public_ips` - The list of public IPs of the server
+        - `id` - The ID of the IP
+        - `address` - The address of the IP
```

Comment:
Do we know if it is going to be a v6 or v4 address?

Reply:
It is both